docs/README.md

```diff
@@ -7,6 +7,7 @@ This folder contains documents that relate to the project.
 ## General
 
 - [Terminology](./terminology.md)
+- [Examples](./../examples/index.md)
```

Comment:
Hmm, probably `README.md` would be a better name for the document as GitHub displays it by default in a given directory